src/server/editorServices.ts

```diff
@@ -988,6 +991,7 @@ namespace ts.server {
 
             const compilerOptions = convertCompilerOptions(projectCompilerOptions);
             const watchOptions = convertWatchOptions(projectCompilerOptions);
+            const typeAcquisition = this.hostConfiguration.typeAcquisition;

```

Comment:
You should combine properties of typeAquisition into compilerOptions in protocol like what watchOptions do.. And have method like convertTypeAquisition to get them from protocol.CompilerOptions

Reply:
Assuming I've done this correctly, it seems like it could be confusing to have the `typeAcquisition.enable` property mixed in with the rest of the compiler options. Does this warrant changing the name of the option to something else?